scripts/Enemy.gd

```diff
@@ -12,6 +12,8 @@ var _motion = Vector2()
 func _ready():
     add_to_group("enemies")
 
+    $Area2D.connect("body_entered", self, "_on_body_entered")
```

Comment:
Not really related to this PR, but we currently use two styles of connecting signals. One using the editor and the other by calling `connect`.  I searched a bit and didn't find any best practices which to use when (obviously calling `connect`in code is more flexible).

The only benefit of connecting signals in the editor instead of code is that that the connection is visible in the editor... but the `signals` tab is really obscure and I almost never open it. Personally I wouldn't be against just always connecting in code, since that is usually what I look at. Thoughts? (If you agree let's create an issue to convert the rest)

Reply:
Agreed. I'll change the editor one in a PR